From ee70e954c6bde2f43988e78218097cd9948cf9f8 Mon Sep 17 00:00:00 2001
From: Pardha Saradhi K <pardha.saradhi.kesapragada@intel.com>
Date: Fri, 4 Mar 2016 12:15:51 +0530
Subject: [PATCH 1889/2508] ASoC: Intel: Skylake: Enable PM ops

commit 6c03b8285800b6e2cd6b814985773facfbc1601e from
https://github.com/01org/linux-apollolake-i

PM ops for skylake and other variants has been accidentally
disabled by CNL patches.This patch re-enables the same

Change-Id: I76f0fa50f9e751b4774eefce46c20b4d93b5cc85
Signed-off-by: Pardha Saradhi K <pardha.saradhi.kesapragada@intel.com>
Reviewed-on: https://git-gar-1.devtools.intel.com/gerrit/8334
Reviewed-by: Babu, Ramesh <ramesh.babu@intel.com>
Tested-by: Babu, Ramesh <ramesh.babu@intel.com>
---
 sound/soc/intel/skylake/skl.c |    2 --
 1 files changed, 0 insertions(+), 2 deletions(-)

diff --git a/sound/soc/intel/skylake/skl.c b/sound/soc/intel/skylake/skl.c
index 40c28eb..5fbac90 100644
--- a/sound/soc/intel/skylake/skl.c
+++ b/sound/soc/intel/skylake/skl.c
@@ -783,9 +783,7 @@ static struct pci_driver skl_driver = {
 	.probe = skl_probe,
 	.remove = skl_remove,
 	.driver = {
-#if 0
 		.pm = &skl_pm,
-#endif
 	},
 };
 module_pci_driver(skl_driver);
-- 
1.7.5.4

